_posts/2020-06-10-community-update.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: "Flink Community Update - June'20"
+date: 2020-06-10T08:00:00.000Z
+categories: news
+authors:
+- morsapaes:
+  name: "Marta Paes"
+  twitter: "morsapaes"
+
+excerpt: And suddenly it’s June. The previous month has been calm on the surface, but quite hectic underneath — the final testing phase for Flink 1.11 is moving at full speed, Stateful Functions 2.1 is out in the wild and Flink has made it into Google Season of Docs 2020.
+---
+
+And suddenly it’s June. The previous month has been calm on the surface, but quite hectic underneath — the final testing phase for Flink 1.11 is moving at full speed, Stateful Functions 2.1 is out in the wild and Flink has made it into Google Season of Docs 2020. 
+
+To top it off, a piece of good news: [Flink Forward](https://www.flink-forward.org/global-2020) is back on October 19-22 as a free virtual event!
+
+{% toc %}
+
+# The Past Month in Flink
+
+## Flink Stateful Functions 2.1 Release
+
+It might seem like [Stateful Functions 2.0 was announced](https://flink.apache.org/news/2020/04/07/release-statefun-2.0.0.html) only a handful of weeks ago (and it was!), but the Flink community has just released Stateful Functions 2.1! This release introduces two new features: state expiration for any kind of persisted state and support for UNIX Domain Sockets (UDS) to improve the performance of inter-container communication in co-located deployments; as well as other important changes that improve the overall stability and testability of the project. You can read the [announcement blogpost](https://flink.apache.org/news/2020/06/09/release-statefun-2.1.0.html) for more details on the release!
+
+As the community around StateFun grows, the release cycle will follow this pattern of smaller and more frequent releases to incorporate user feedback and allow for faster iteration. If you’d like to get involved, we’re always [looking for new contributors](https://github.com/apache/flink-statefun#contributing) — especially around SDKs for other languages (e.g. Go, Rust, Javascript).
+
+<hr>
+
+## Testing is ON for Flink 1.11
+
+Things have been pretty quiet in the Flink community, as all efforts shifted to testing the newest features shipping with Flink 1.11. While we wait for the first Release Candidate (RC) to be out, you can check the progress of testing in [this JIRA burndown board](https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=364&projectKey=FLINK) and learn more about some of the [upcoming features](https://flink.apache.org/news/2020/05/07/community-update.html#warming-up-for-flink-111) in these Flink Forward videos:
```

Comment:
The first RC is out for quite a while: https://lists.apache.org/thread.html/r4952ccfd5f5562839f14b6e724f722b1302389c4b32c61db59ce9b31%40%3Cdev.flink.apache.org%3E

Reply:
It's not a "true" RC in the sense of a precursor of the release, at least if I understand correctly.